PostBy/View Controllers/DetailsViewController.m

```diff
@@ -152,10 +164,9 @@ - (void) setButtonsUserInteractionTo:(BOOL)value {
 }
 
 - (IBAction)editPost:(id)sender {
-    // TODO: Implement EDIT feature
+    [self performSegueWithIdentifier:@"EditPostSegue" sender:self.postVM];
 }
 
-
 - (IBAction)showPostLocation:(id)sender {
     [self performSegueWithIdentifier:@"DetailsShowMap" sender:self.postVM];
```

Comment:
To prevent inconsistence caused by typo between line 171 and line 175, it is better to create a constant for it.

Reply:
The same for line 178.